examples/object_detection/android-jetpack-compose/app/build.gradle

```diff
@@ -0,0 +1,91 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.example.objectdetection'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.example.objectdetection"
+        minSdk 28
```

Comment:
Is there anything restricting the minimum SDK to 28 instead of 24?

Reply:
nope, I might have needed it to try something at some point and forgot to change it back. It's set to 24 now